fetch/fetchRequest.js

```diff
@@ -12,12 +12,22 @@
 
 var fetch = require('cross-fetch');
 
+/* eslint-disable complexity */
 function fetchRequest(method, url, args) {
+  var body = args.data;
+  var headers = args.headers || {};
+  var contentType = (headers['Content-Type'] || headers['content-type'] || '').toLowerCase();
```

Comment:
The extra `toLowerCase()` isn't needed here. We only need to assert that the JSON key is case sensitive.